### DISCUSSION TITLE
How do you get the older `edge` versions of AdGuard Home?

### DISCUSSION THREAD

**Discussion by flavorgold1** [AdguardTeam/AdGuardHome#3861] [Q&A] [answered]
I'm trying to download the `edge` version of AdGuard Home from before this one.

**Comment by TheCeet** [comment]
‘Assets’ at the bottom from an release. 
https://github.com/AdguardTeam/AdGuardHome/releases

**Comment by flavorgold1** [comment]
> ‘Assets’ at the bottom from an release. https://github.com/AdguardTeam/AdGuardHome/releases

That's `beta`, not `edge`.

**Comment by TheCeet** [comment]
> > ‘Assets’ at the bottom from an release. https://github.com/AdguardTeam/AdGuardHome/releases
> 
> That's `beta`, not `edge`.

Oops. My bad!
I think it’s not possible. But the devs will answer tomorrow

**Comment by ameshkov** [comment, marked as answer]
I think the only way to get an older edge version is to compile it manually.
